iotc_base_container/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT license.
 #-------------------------------------------------------------------------------
 
-FROM  ubuntu:xenial
+FROM  ubuntu:bionic
```

Comment:
is this must?

Reply:
how this would affect the other packages?

Reply:
Let me verify.

Reply:
It seems to not be necessary. I can remove it, it just updates it from 16.04 to 18.04 ubuntu. I have no idea if it would affect other packages, good point.